src/client/components/movies/Movies.js

```diff
@@ -27,19 +32,24 @@ class Movies extends React.Component {
       )
     }
     if (this.props.movies.length) {
-      const renderMovie = (movie) => { return <Movie key={movie.id} movie={movie} /> }
+      const renderMovie = (movie) => { 
```

Comment:
please move it out of scope of render method. and also move `MovieContainer = withIdConnection(Movie)` out of scope component